modules/dlm/src/main/java/org/elasticsearch/dlm/action/TransportExplainDataLifecycleAction.java

```diff
@@ -72,49 +77,52 @@ protected void masterOperation(
         ExplainDataLifecycleAction.Request request,
         ClusterState state,
         ActionListener<ExplainDataLifecycleAction.Response> listener
-    ) throws Exception {
+    ) {
+        // Empty array to force cluster privilege check, but no need for index privileges checks
+        // TODO this is hacky, and we won't do this in the final implementation
+        privilegesCheck.checkCanConfigure(Strings.EMPTY_ARRAY, ActionListener.wrap(ignored -> {
```

Comment:
This call  to `checkCanConfigure` is the only change here; all code inside the action listener wrap block is as before